tests/Festung/FrontendSpec.hs

```diff
@@ -372,6 +376,25 @@ spec = withApp_ $ do
                         , rowsChanged = -1
                         }
 
+        it "Binds bulk parameters" $ do
+            createTableWithData "/vault" password
+            postJson "/vault" password $
+                BulkParametrizedQuery "INSERT INTO foo(bar) VALUES (?)" [[RInt 3], [RInt 4], [RInt 5]]
+            statusIs 200
```

Comment:
The previously existing test cases didn't check that either, but I think we should verify that the response body contains the correct values. I did a manual test and inserted 3 rows with a bulk insert like so
```http
$ http localhost:2728/1 Authorization:Zm9vCg== sql='CREATE TABLE foo (id INT, b VARCHAR)' params:='[]'
{
    "data": [],
    "headers": [],
    "last_row_id": 0,
    "rows_changed": 0
}
$ http localhost:2728/1 Authorization:Zm9vCg== sql='INSERT INTO foo VALUES (?, ?)' bulkParams:='[[1, "a"],[2,"b"],[3,"c"]]'
{
    "data": [],
    "headers": [],
    "last_row_id": 3,
    "rows_changed": 1
}
```
As you can see the `"last_row_id"` is correct, but the `"rows_changed"` should also be set to 3.